common/ostream.h

```diff
@@ -66,7 +66,7 @@ void PrintTo(T* p, std::ostream* out) {
 namespace llvm {
 
 // Injects an `operator<<` overload into the `llvm` namespace which detects LLVM
-// types with `raw_ostream` overloads and uses that to map to a `std::ostream`
+// types with `raw_ostream` overloads and use that to map to a `std::ostream`
```

Comment:
I'm not the best at English grammar, but I think the original is correct here?

```suggestion
// types with `raw_ostream` overloads and uses that to map to a `std::ostream`
```

Reply:
But the subject here is plural i.e. types

Reply:
I agree with @chandlerc that the original is correct. "Injects X and uses Y." -- I think the sentence is intended to parse differently than you're reading it.

Reply:
@jonmeow I see :thinking:.